fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -309,6 +312,7 @@ public FHIRPersistenceJDBCImpl(Properties configProps, IConnectionProvider cp, F
 
         this.cache = cache;
         this.payloadPersistence = null;
+        this.searchHelper = new SearchUtil();
```

Comment:
I was wondering if you think that SearchUtil should be an interface...in which case this helper would need to be injected.

Reply:
Perhaps that's a second level of refactor

Reply:
yeah, it gets injected for the running server.  this particular constructor is only used from tests and so I didn't both to have them all inject it

Reply:
I did debate making an interface for SearchUtil and put it in fhir-server-spi (similar to the one for FHIRRestHelper), but that was more to try avoiding a dependency on fhir-search within our custom operation modules.
In the end, I didn't bother because those modules will mark all their dependencies as "provided" anyway.